solutions/bio1/RomanLookAndSay/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { SCode, SCodeBlock, SText, STitle } from "components";
+
+export const RomanLookAndSay = () => {
+  return (
+    <>
+      <STitle>Idea</STitle>
+      <SText>
+        Since constraints are small, this is a simple simulation question. 
+      </SText>
+      <STitle>Solution</STitle>
+      <SText> 
+        We can solve the problem by applying the "Roman look-and-say" description {<SCode>n</SCode>} times, and then outputting the number of Is and Vs in the string after {<SCode>n</SCode>} operations. 
```

Comment:
Can n be in latex ($$) rather than in code blocks. Also Is and Vs needs to be 'I's and 'V's

Reply:
actually I think it should be $I$-s and $V$-s.

On another note - how do you and Boris make these quote things? I haven't found the button for it lol

Reply:
When you go into the code you can hover over the line number and a plus thing comes up

Reply:
If you click files changed, then hover on a bit of code there's a plus icon which you can click. From there you can either write a single comment or a full review

Reply:
cool :) thanks